### ISSUE TITLE
works on intel not risc

### ISSUE THREAD

**Issue by CodeAKrome** [BerriAI/litellm#334]
What's the problem? (if there are multiple - list as bullet points)
Expected same results on Intel/Ubuntu and Rpi4b/Kali GNU/Linux Rolling aarch64 platforms.
On intel it works, not so on rpi.

```python
from litellm import completion
import os
import sys

if len(sys.argv) > 1:
    model = "huggingface/" + sys.argv[1]
else:
    model = "huggingface/cerebras/btlm-3b-8k-base"

print(f"model:\t{model}\n")
messages = [{ "content": "Hello, how are you?","role": "user"}]
response = completion(model, messages=messages)
print(f"{type(response)}")
print(response)
```

# Intel
```
model:	huggingface/PygmalionAI/metharme-1.3b

<class 'litellm.utils.ModelResponse'>
{
  "choices": [
    {
      "finish_reason": "stop",
      "index": 0,
      "message": {
        "content": "Hello, how are you?\n\nI'm good, how are you?\n\nI'm",
        "role": "assistant",
        "logprobs": null
      }
    }
  ],
  "created": 1694489382.34404,
  "model": "PygmalionAI/metharme-1.3b",
  "usage": {
    "prompt_tokens": 6,
    "completion_tokens": 16,
    "total_tokens": 22
  }
}
```

# Rpi4b

```
model:	huggingface/PygmalionAI/metharme-1.3b

Traceback (most recent call last):
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/main.py", line 559, in completion
    model_response = huggingface_restapi.completion(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/llms/huggingface_restapi.py", line 108, in completion
    raise HuggingfaceError(
litellm.llms.huggingface_restapi.HuggingfaceError: The following `model_kwargs` are not used by the model: ['details'] (note: typos in the generate arguments will also show up in this list)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kyle/srcLocal/llm/./lite-llm.py", line 13, in <module>
    response = completion(model, messages=messages)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/utils.py", line 584, in wrapper
    raise e
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/utils.py", line 544, in wrapper
    result = original_function(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/timeout.py", line 44, in wrapper
    result = future.result(timeout=local_timeout_duration)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/concurrent/futures/_base.py", line 456, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/timeout.py", line 33, in async_func
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/main.py", line 870, in completion
    raise exception_type(
          ^^^^^^^^^^^^^^^
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/utils.py", line 1959, in exception_type
    raise e
  File "/home/kyle/srcLocal/llm/.venv/lib/python3.11/site-packages/litellm/utils.py", line 1808, in exception_type
    raise InvalidRequestError(
litellm.exceptions.InvalidRequestError: HuggingfaceException - The following `model_kwargs` are not used by the model: ['details'] (note: typos in the generate arguments will also show up in this list)
```

**Comment by ghost** [comment]
Hey @CodeAKrome that looks like an old version - can you update to the latest version and let me know if it works?

**Comment by ghost** [comment]
For context the error was being caused by huggingface's default inference endpoint not accepting the 'details' arg but their TGI endpoints accepting it. 

This has been fixed since v0.1.600 onwards - bab36c2c6fb4a25b4e4fd984403c6071c6c0b0f0

**Comment by ghost** [comment]
related issue - https://github.com/BerriAI/litellm/issues/332

**Comment by ghost** [comment]
@CodeAKrome closing for now as this has been fixed from v0.1.600 onwards. Please re-open if bumping version doesn't work for you.

**Comment by CodeAKrome** [comment]
Successfully installed litellm-0.1.749 tokenizers-0.14.0
lite-llm.py PygmalionAI/metharme-1.3b "Hello, how are you?"
```
litellm.exceptions.InvalidRequestError: HuggingfaceException - The following `model_kwargs` are not used by the model: ['details'] (note: typos in the generate arguments will also show up in this list)
```

```python
#!env python
from litellm import completion
import os
import sys

if len(sys.argv) > 1:
    model = "huggingface/" + sys.argv[1]
    text = sys.argv[2]
else:
    model = "huggingface/cerebras/btlm-3b-8k-base"
    text = "Hello, how are you?"

print(f"model:\t{model}\n")
messages = [{ "content": text,"role": "user"}]
response = completion(model, messages=messages)
print(f"{type(response)}")
print(response)
```
